test/functional/apps/dashboard_elements/controls/control_group_settings.ts

```diff
@@ -50,7 +50,6 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         const growSwitch = await testSubjects.find('control-editor-grow-switch');
         expect(await growSwitch.getAttribute('aria-checked')).to.be('true');
         await testSubjects.click('control-editor-cancel');
-        await testSubjects.click('confirmModalConfirmButton');
```

Comment:
I fixed the logic for deciding whether or not to show the confirm modal so, since we weren't actually making any changes in this test and instead we were just checking some values, the confirm modal no longer shows up (which is actually the expected behaviour).